doc/changelog/11-infrastructure-and-dependencies/10471-ocaml-408.rst

```diff
@@ -0,0 +1 @@
+- Coq now officially supports OCaml 4.08.
```

Comment:
Please be more exhaustive: it could be compiled with OCaml 4.08 before, but with warnings. Mention the PR number and yourself as the author of it.